lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
we should be using `res.writeHead()` and setting the current status code, which may or may not be `200`